Userland/Games/GameOfLife/Pattern.h

```diff
@@ -14,13 +14,13 @@
 
 class Pattern final {
 public:
-    Pattern(Vector<DeprecatedString>);
-    Vector<DeprecatedString> pattern() { return m_pattern; };
+    Pattern(Vector<StringView>);
+    Vector<String> pattern() { return m_pattern; };
     GUI::Action* action() { return m_action; }
     void set_action(GUI::Action*);
     void rotate_clockwise();
 
 private:
     RefPtr<GUI::Action> m_action;
-    Vector<DeprecatedString> m_pattern;
+    Vector<String> m_pattern;
```

Comment:
I think a vector of `String` is a bit more than what's needed here :smile:, the `Pattern` class could be replaced with something very like the `CharacterBitmap`, but instead it owns its data. 

So basically something like:
```c++
class Pattern final
{
    Pattern(StringView pattern, Gfx::IntSize size)
      : m_pattern(pattern), m_size(size)

   char at(int x, int y) const {
        return m_pattern[y * m_size.width() + x];
   }

   // ...
private:
    Vector<char> m_pattern;
    Gfx::IntSize m_size;
    RefPtr<GUI::Action> m_action;
};
```
```c++
Pattern diehard {
    "......O."
    "OO......"
    ".O...OOO"sv,
    {8, 3}
};
```

Reply:
Recreated the Pattern class but left out the IntSize member. A reference to LibGfx seems odd to me if IntSize is the only reason LibGfx is used in here.

Reply:
Looks good :+1: